src/pages/solutions/for-government.mdx

```diff
@@ -55,46 +55,28 @@ import accessiblePhoto from '@images/solutions/accessible.svg';
 
   </Section>
-  <Section title="Application Development">
+  <Section title="Let us build">
     <Image
       slot="image"
       src={applicationDevelopmentPhoto}
       alt="a person standing in front of a monitor holding a wrench with a crane in the background"
       class="w-full max-w-[400px]"
     />
 
-    - Custom development. We can re-imagine your entire business process on the web carefully including the advantages of GIS. Examples
-    - Configurable apps
-    - Low code no code
+    Once your data is accessible, take it to the next level by integrating it into your agency’s important business processes, making it visible and tangible in an interactive web mapping experience, or almost anything else you can imagine. UGRC works in both the low code/no code and traditional development spaces to build custom tools, processes, and applications for state government agencies to unlock the power of location. We can work with you to integrate spatial data and GIS functionality to complement existing workflows; improve and streamline business processes from beginning to end; or imagine new processes and pathways to achieve your goals. If you have access to, or are, an application developer, check out our offerings that are free and easy to use.
 
   </Section>
-  <Section title="Imagine it" stripe>
+  <Section title="Investigate your data" stripe>
     <Image
-      slot="image"
-      src={imaginationPhoto}
-      alt="two people placing puzzle pieces together in the shape of a road leading to a lightbulb"
-      class="w-full max-w-[400px]"
-    />
-
-    - UGRC can help you find service areas for state buildings.
-    - How many citizens can be served based on a 30 minute drive time
-    - find my cash reporting,
-    - corona virus work?
-
-  </Section>
-  <Section title="Data science">
-    <Image
-      slot="image"
+      slot="investigate"
       src={dataSciencePhoto}
       alt="beakers and a microscope"
       class="w-full max-w-[400px]"
     />
 
-    - UGRC currently aggregates parcels, address points, roads data from counties in support of 911, Taxation, and Elections
-    - We can work together to aggregate data from your agency
+    Government agencies collect massive amounts of data. Perhaps your data needs a boost, or to be viewed in a totally different perspective. UGRC uses creative and innovative data science methods and technology to reveal hidden connections within your data, enabling smarter, data-driven decisions. Allow us to help you analyze, summarize, transform, and enrich your data with meaningful location information to illuminate new priorities and solutions, answer questions, and optimize resources.
 
-  </Section>
```

Comment:
@steveoh I accidentally deleted this. Can it be added back to this commit rather than deleting the commit and starting again?

Reply:
I added it back for you.